### ISSUE TITLE
Create a container registry on AWS

### ISSUE THREAD

**Issue by lopopolo** [artichoke/project-infrastructure#8]
For hosting the CI container: https://github.com/artichoke/artichoke-ci

**Comment by lopopolo** [comment]
Blocked on GH-3.

**Comment by lopopolo** [comment]
GitHub actions obsoletes this ticket. Close won't fix.
